x-pack/test/apm_api_integration/common/utils/round_number.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { round } from 'lodash';
+
+/**
+ * To avoid floating precision problems, rounds any number value to max 5 decimals
```

Comment:
We can also use `parseFloat(number.toPrecision(5))`. It might be better for really low values (e.g., 0.000001), and would also work for really high values.